pkg/models/datasource_cache_test.go

```diff
@@ -129,16 +129,21 @@ func TestDataSourceProxyCache(t *testing.T) {
 
 		json := simplejson.New()
 		json.Set("tlsAuthWithCACert", true)
+		json.Set("serverName", "server-name")
 
 		tlsCaCert, err := util.Encrypt([]byte(caCert), "password")
 		So(err, ShouldBeNil)
 
+		So(err, ShouldBeNil)
```

Comment:
Looks to me as if this error check is left over?

```suggestion
```

Reply:
Not sure how that happened. I'll fix and push!